.github/workflows/updatecli.yml

```diff
@@ -0,0 +1,42 @@
+name: "Updatecli: Dependency Management"
+
+on:
+  schedule:
+    # Run every hour
+    - cron: '0 * * * *'
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  issues: write
+  pull-requests: write
+
+jobs:
```

Comment:
Confused on this, we don't have actions available in rancherlabs ? Or is this using our runners somehow.

Reply:
Since the repo is public, we can use GH Actions for free.